sunpy/net/fido_factory.py

```diff
@@ -102,9 +102,19 @@ def __getitem__(self, aslice):
 
     def get_response(self, i):
```

Comment:
Do tests exist for this already?

We should add a test to make sure it works for each client.

Reply:
This method existed earlier. Now question is, should I deprecate/remove its old functionality?
That allowed to retrieve responses based on index. Currently both things works in my PR.
Or should I add a new method instead of `get_response` like `fido_results.view('hek')` to do this?

Reply:
Would there be a reason to still need to use an index?

Reply:
See https://github.com/sunpy/sunpy/pull/4358#discussion_r475250359